src/SDKs/SqlManagement/Sql.Tests/Utilities/SqlManagementTestUtilities.cs

```diff
@@ -40,6 +40,8 @@ public class SqlManagementTestUtilities
 
         public const string DefaultEuapPrimaryLocation = "East US 2 EUAP";
 
+        public const string DefaultEuapPrimaryLocationId = "eastus2euap";
+
         public const string DefaultLogin = "dummylogin";
```

Comment:
@dnayantara please push all these information via test connection string, rather than hard coding it in code.
Refer to the below link to push custom data during tests
https://github.com/Azure/azure-powershell/blob/preview/documentation/Using-Azure-TestFramework.md#environment--custom